### DISCUSSION TITLE
How to make custom music?

### DISCUSSION THREAD

**Discussion by Samdurp** [jazz-soft/react-midi-player#1] [General]
Just wondering how I can make my own chunk of text that resembles music. The massive Base64 string is unreadable to human eyes and I'm wondering how I can get this software to play anything other than Mary Had A Little Lamb. Is there a key somewhere? A way to translate from the Base64 string to something that can be understood? A direct way to show what characters in that string mean? I assume it's delimited by the forward slashes somehow, but I have been unsuccessful in finding out how exactly any of the data is written or read.

Thank you for your time, this software looks great and I hope to be able to use it!

**Comment by jazz-soft** [comment]
The "massive Base64 string" is an encoded MIDI file.
There are tons of MIDI files available for download, and there are lots of software that would let you create your own MIDI files.
See, for example, https://github.com/jazz-soft/JZZ-midi-SMF
